daprdocs/content/en/operations/extending/developing-grpc-components.md

```diff
@@ -6,8 +6,45 @@ weight: 250
 description: "Extending Dapr with external gRPC-based components"
 ---
 
-[gRPC-based](https://grpc.io/) Dapr components are typically run as containers or processes that communicate with the Dapr main process via [Unix Domain Sockets](https://en.wikipedia.org/wiki/Unix_domain_socket).
+## Components and Pluggable Components
```

Comment:
looks like that both terms are interchangeably being used, can we name this section as just `Pluggable Components Overview` ?